### PR TITLE
Removed some currently unnecessary permissions from AndroidManifest.xml

### PR DESCRIPTION
**Pull request by RohanTalip**
## What does this PR accomplish?

It removes some currently unnecessary permissions from AndroidManifest.xml

See issue #56 

## Did you add any dependencies?

No.

## How did you test the change?

I built the app locally, and ran it in the Android emulator.

### CODE REVIEW

**Comment by RohanTalip** [comment]
I left the location permissions in the file for now, as it was less clear that these need to be removed, given eventually the app may want to access the device's location.

File: client/ionic/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -46,19 +46,8 @@
...
     <uses-permission android:name="android.permission.INTERNET" />
-    <!-- Camera, Photos, input file -->
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <!-- Geolocation API -->
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-feature android:name="android.hardware.location.gps" />
-    <!-- Network API -->
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```
Comment (epicfaace) [inline client/ionic/android/app/src/main/AndroidManifest.xml]: Why don't we need ACCESS_NETWORK_STATE?

Comment (myiremark) [reply client/ionic/android/app/src/main/AndroidManifest.xml]: Also, for the first version, do we actually need access location?

Comment (shankari) [reply client/ionic/android/app/src/main/AndroidManifest.xml]: @epicfaace `ACCESS_NETWORK_STATE` allows you to control whether to access the internet only when connected to WiFi or sth like that. Typically not used for foreground operation, and v0.1 app is all foreground

@myiremark I believe the answer at the standup meeting was "no"

**Comment by brunobowden** [comment]
@RohanTalip - thanks for cleaning this up

File: client/ionic/android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (RohanTalip) [reply client/ionic/android/app/src/main/AndroidManifest.xml]: My understanding of ACCESS_NETWORK_STATE is that it is needed if you want to check if you are connected to a network or not, i.e. not likely needed at present.  The Ionic code I looked at today only seems to use it in the [Network API](https://github.com/ionic-team/capacitor/blob/master/site/docs-md/apis/network/index.md).  The other reference I found was in the [(Ionic) Capacitor Android Plugin Guide
](https://github.com/ionic-team/capacitor/blob/master/site/docs-md/plugins/android.md).

I'll comment out the location access for now in another commit, coming shortly.

Comment (RohanTalip) [reply client/ionic/android/app/src/main/AndroidManifest.xml]: The location access permissions have been commented out for now.

**Comment by advayDev1** [comment]
Thanks for the PR!  Per our discussion this morning, we are committed to Flutter for the client, so closing these out for now.

**Pull request closed without merging**